lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/dllmain.cpp

```diff
@@ -1353,11 +1376,18 @@ void eos_create(EOSConfig& eosConfig)
         steamApiInterfaceVersionsAsCharArray.push_back('\0');
 
         steam_platform.SteamApiInterfaceVersionsArray = reinterpret_cast<char*>(steamApiInterfaceVersionsAsCharArray.data());
+                
+        auto size = steamApiInterfaceVersionsAsCharArray.size();
+
+        if (size > (std::numeric_limits<uint32_t>::max)()) {
+            log_error("Size given for SteamApiInterfaceVersionsAsCharArray exceeds the maximum value for uint32_t.");
+            throw std::overflow_error("Size exceeds the maximum value for uint32_t");
```

Comment:
I'm intrigued this worked because I thought we had exceptions disabled.

Reply:
I wrote this without knowing that they were disabled, just seemed like the appropriate implementation just looking at the narrow bit of code here.

Is it possible that exceptions are disabled in the sense that throwing them does not actually _do_ anything?

Either way, should the implementation be changed so that a default value is used instead of an exception being thrown like it currently is?

What _would have_ happened previously if there was an overflow error?

Reply:
Steps to resolve the concerns described in this conversation have been articulated in a separate conversation, so I am marking this one as resolved.